static/app/views/eventsV2/table/tableView.tsx

```diff
@@ -183,11 +184,26 @@ class TableView extends Component<TableViewProps & WithRouterProps> {
       });
 
       return [
-        <Tooltip key={`eventlink${rowIndex}`} title={t('View Event')}>
-          <StyledLink data-test-id="view-event" to={target}>
-            {value}
-          </StyledLink>
-        </Tooltip>,
+        organization.features.includes('discover-quick-context') ? (
+          <QuickContextHoverWrapper
+            key={`quickContextEventHover${rowIndex}`}
+            dataRow={dataRow}
+            contextType={ContextType.EVENT}
+            organization={organization}
+          >
+            <Tooltip key={`eventlink${rowIndex}`} title={t('View Event')}>
+              <StyledLink data-test-id="view-event" to={target}>
+                {value}
+              </StyledLink>
+            </Tooltip>
+          </QuickContextHoverWrapper>
```

Comment:
We can probably extract this into a variable so we don't have to repeat it

Reply:
Done!